packages/zoltan2/core/src/algorithms/color/Zoltan2_AlgHybridD1-2GL.hpp

```diff
@@ -0,0 +1,357 @@
+#ifndef _ZOLTAN2_DISTANCE1_2GHOSTLAYER_HPP_
```

Comment:
Comments regarding atomics from Zoltan2_AlgHybridD2.hpp apply to this file as well.